tests/cypress/integration/pdv_hoc_site_spec.js

```diff
@@ -0,0 +1,487 @@
+describe('Test H.O.C.', () => {
+
+  it('Coveo search', () => {
+    // out of scope
+    // not testable in local environments
+  })
+
+  it('Google Analytics', () => {
+    // out of scope
+    // not testable in local environments
+  })
+
+  it('Qualtrics feedback', () => {
+    // out of scope
+    // not testable in local environments
+  })
+
+  it('Classic/V1 release notes (en-US)', () => {
+    cy.visit('en-us/Content/Topics/ReleaseNotes/okta-relnotes.htm')
+
+  })
+
+  it('OIE release notes (en-US)', () => {
+    cy.visit('oie/en-us/Content/Topics/ReleaseNotes/oie-relnotes.htm')
+
+  })
+
+  it('Product documentation landing page (en-US)', () => {
+    cy.visit('en-us/Content/index.htm')
+    // Logo image
+    cy.get('img[title="Okta Support"]')
+      .should('be.visible')
+      .and(($img) => {
+        expect($img[0].naturalWidth).to.equal(382)
+      })
+    // Copyright
+    cy.get('p.copyright')
+      .contains('©')
+    // Menu dropdown
+    cy.get('div.navbar > div.dropdown')
+      .should(($div) => {
+        expect($div).to.have.length(5)
+      })
+    cy.get('div.dropdown-content')
+      .eq(0)
+      .should('be.hidden')
+      .invoke('show')
+    cy.get('div.dropdown-content > div > a')
+      .should(($a) => {
+        expect($a).to.have.length(9)
+        expect($a).to.have.attr('href')
+      })
+    // All eight tiles on landing page have links
```

Comment:
this comment is for the line statement above right? We should move it to line 48 (before we test it) or line 49

Reply:
Sorry, I missed this one yesterday.
Actually, this comment applies to the code that immediately follows. But, I didn't comment these similar calls consistently elsewhere (such as above the old line 50), so I see how that could be... ambiguous.
I can fix this by applying comments for these assertions, to make it clearer what we're actually checking.